.github/workflows/tests-integration.yml

```diff
@@ -0,0 +1,22 @@
+name: Run Unit Tests
```

Comment:
`Run Integration Tests`

Reply:
Thanks! I've made the label change and also added a `docker-compose up` to `tests-integration` -- they now work. Thanks @markpatton !